src/components/App/CacheBuster.js

```diff
@@ -0,0 +1,47 @@
+import { useEffect } from 'react'
+import { useSelector } from 'react-redux'
+import { selectLocation } from 'selectors'
+import packageJson from '../../../package.json'
+import { refreshCacheAndReload, execute } from 'actions'
+global.appVersion = packageJson.version
+
+export default function CacheBuster(props) {
+	const location = useSelector(selectLocation) || {}
+	const semverGreaterThan = (versionA, versionB) => {
+		const versionsA = versionA.split(/\./g)
+
+		const versionsB = versionB.split(/\./g)
+		while (versionsA.length || versionsB.length) {
+			const a = Number(versionsA.shift())
+
+			const b = Number(versionsB.shift())
+			// eslint-disable-next-line no-continue
+			if (a === b) continue
+			// eslint-disable-next-line no-restricted-globals
+			return a > b || isNaN(b)
+		}
+		return false
+	}
+
+	useEffect(() => {
+		fetch('/meta.json')

```

Comment:
can you add some  unique random query param every time you fetch, just in case of cache